### ISSUE TITLE
CreationView should redirect to ShowView when EditionView is disabled

### ISSUE THREAD

**Issue by ramSeraph** [marmelab/ng-admin#393]
CreationView upon success redirects to EditionView, but if EditionView is disabled this goes nowhere.

In this case it should try and redirect to ShowView or ListView which ever is still available

**Comment by fzaninotto** [comment]
Let me rephrase this: For a particular entity, you want to authorize creation, but not edition. Could you share the use case that leads to this weird decision?

**Comment by ramSeraph** [comment]
There is only one field on the entity which is editable and even that is just a toggle. I would prefer if that can just be toggled by a custom button in the list view( this I have already implemented ).

Creation on the other hand has a lot more fields and does call for a separate view.

Even otherwise, there will definitely be use cases where someone might want to just create and delete and not edit an entity. Automatically redirecting to an empty edit view might be undesired in these cases as well.

**Comment by fzaninotto** [comment]
Again, can you explain me the domain leading to such decision?

**Comment by ramSeraph** [comment]
Ah, you meant the domain, It is the admin page for a social app and the entities are related to the invite codes for the same.

**Comment by ramSeraph** [comment]
Thanks for the quick turnaround. Checked and it works
